cmd/kubeadm/app/constants/constants.go

```diff
@@ -37,10 +37,12 @@ const (
 	APIServerCertAndKeyBaseName = "apiserver"
 	APIServerCertName           = "apiserver.crt"
 	APIServerKeyName            = "apiserver.key"
+	APIServerCommonName         = "kube-apiserver"
```

Comment:
Please add a comment for all these three new constants. It wasn't obvious to me what they were used for